README.md

```diff
@@ -42,6 +42,14 @@ make run
 make run-backend 
 ```
 
+### Expose saleor-app-checkout
+```shell
+make tunnel
+```
+
+> Note
+> `saleor-app-checkout` needs to be installed manually in Saleor Dashboard
```

Comment:
Actually you can do it from script

```
$saleor app tunnel -h

saleor app tunnel [port]

Expose your Saleor app remotely via tunnel

Positionals:
  port                                                  [number] [default: 3000]

Options:
      --json             Output the data as JSON                       [boolean]
      -u, --instance, --url                                                 [string]
      --name             The application name for installation in the Dashboard [string]
      --force-install                                 [boolean] [default: false]
       -V, --version          Show version number                           [boolean]
       -h, --help             Show help                                     [boolean]
```

Reply:
@2can @zaiste is it possible?

Reply:
@witoszekdev you need two tunnels set up to make it work. One for the Saleor API and another for the Saleor app. cc @krzysztofwolski

Reply:
@zaiste The issue here is that we need the app to be installed in local Saleor (**not** cloud) that's running in Docker, when I tried to use `--url=localhost:8000` it didn't work

Reply:
@lkostrowski Here's what we need to do to be able to install apps locally:
- Saleor Core must be exposed via tunnel (e.g. ngrook, our CLI tunnel is only meant for apps at the moment)
- Saleor App Checkout must be exposed via CLI's tunnel

We can pass the instance URL to the CLI to install Saleor App Checkout inside Saleor and to tunnel it at the same time.

This setup looks really clunky, but we have to do this to be able to use the app inside Saleor because:
- app needs to be exposed to the Internet to listen to webhooks
- Saleor cannot install apps from localhost
- we need some URL for CLI with instance url
- app needs to communicate with Saleor, our current docker configuration uses `bridge` network which makes it difficult to communicate between services
  - we cannot use `host` network - it only works on Linux
  - we cannot just use localhsot in app service, it means "use localhost from inside this container"

I don't really have a solution to this, so I'll update the docs for now, and we can think of a better DX.

Reply:
@zaiste @witoszekdev what is needed to actually run everything in localhost? I don't think its difficult - can you evaluate?

Reply:
FYI 

@przlada is working on enabling installing localhost apps, when running locally. After that, if we fix Docker networks, full local development should be enabled